src/globals/scss/helpers/_spacing.scss

```diff
@@ -0,0 +1,42 @@
+// Get single spacing unit. We use "tablet" values from settings/spacing as they
+// cascade consistently. However, there is a naming disconnect here as it feels
+// odd to fetch "tablet" values specifically. We should discuss naming in the
+// spacing maps.
+// Arguments:
+// $negative == true returns a negative unit.
+@function govuk-spacing-unit($spacing-map, $negative: false) {
```

Comment:
I'm not sure how I feel about this. It still feels kinda weird to have a scale that's both responsive and non-responsive in this way. And as I commented in the settings file, it introduces a fragile convention that means you always have to set a tablet breakpoint, even if it doesn't change from mobile.

I'm not sure I have a better solution. Other than to be more explicit and have a scale and a responsive scale, or… something.

Reply:
See my comment below :)